poly-synth/src/poly-synth.js

```diff
@@ -74,12 +81,24 @@ class PolySynth {
    * Returns parameters that affect how a voice is constructed.
    * @returns {Object} parameters Parameters which affect the output of a voice
    *                              if set before the voice is constructed.
-   * @returns {Number} parameters.attack Seconds until full amplitude.
-   * @returns {Number} parameters.decay Seconds until sustain level.
-   * @returns {Number} parameters.sustain The steady amplitude of the note as it
-   *                                      is pressed.
-   * @returns {Number} parameters.release Seconds between the release of a note
-   *                                      and zero amplitude.
+   * @returns {Number} parameters.gainAttack Seconds until full amplitude.
+   * @returns {Number} parameters.gainDecay Seconds between full level and
+   *                                        sustain and level.
+   * @returns {Number} parameters.gainSustain The steady amplitude of the note
+   *                                          as it is pressed.
+   * @returns {Number} parameters.gainRelease Seconds between the release of a
+   *                                          note and zero amplitude.
+   * @returns {Number} parameters.filterCutoff The cutoff of the low pass
+   *                                           filter.
+   * @returns {Number} parameters.filterQ The peak of the response at the cutoff
+   *                                      frequency.
+   * @returns {Number} parameters.filterAttack Seconds until full detune.
+   * @returns {Number} parameters.filterDecay Seconds between full detune and
+   *                                          sustain detune.
+   * @returns {Number} parameters.filterSustain The steady detune level as a
+   *                                            note is pressed.
+   * @returns {Number} parameters.filterRelease Seconds betweeen the release of
+   *                                            a note and zero detune.
```

Comment:
Probably want to say the audio params are k-rate not a-rate somewhere around here.